templates/default-layout.lucius

```diff
@@ -153,226 +159,671 @@ html, body {
+  text-decoration: none;
 }
 
-.not-available {
-  background-color: #{Css.notAvailableBackgroundColor};
-  border-color: #{Css.darken20 Css.notAvailableBackgroundColor};
+code, pre {
+  background-color: #{Css.codeBackground};
+  border-radius: 3px;
+  color: #{Css.codeForeground};
+  font-family: Consolas, "Liberation Mono", Menlo, Courier, monospace;
+  font-size: 87.5%;
+}
+
+:target code,
+:target pre {
+  background-color: #{Css.darken5 Css.codeBackground};
+}
+
+code {
+  padding: 0.2em 0;
+  margin: 0;
+  white-space: pre-wrap;
+  word-wrap: break-word;
 }
 
-/* Typography */
-h1, h2 {
+code::before,
+code::after {
+  letter-spacing: -0.2em;
+  content: "\00a0";
+}
+
+a > code {
   font-weight: normal;
 }
 
+a > code::before {
+  content: "🡒";
+  letter-spacing: 0.33em;
+}
+
+a:hover > code {
+  color: #{Css.linkColor};
+}
+
+pre {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 1em 1.25rem; /* Using rem here to align with lists etc. */
+  overflow: auto;
+  white-space: pre;
+  word-wrap: normal;
+}
+
+pre code {
+  background-color: transparent;
+  border: 0;
+  font-size: 100%;
+  max-width: auto;
+  padding: 0;
+  margin: 0;
+  overflow: visible;
+  line-height: inherit;
+  white-space: pre;
+  word-break: normal;
+  word-wrap: normal;
+}
+
+pre code::before,
+pre code::after {
+  content: normal;
+}
+
+h1 {
+  font-size: 3.052em;
+  font-weight: 300;
+  letter-spacing: -0.5px;
+  line-height: 1.125;
+  margin-top: 1.563rem;
+  margin-bottom: 1.25rem;
+}
+
+@media (min-width: 52em) {
+  h1 {
+    font-size: 3.814em;
+    margin-top: 5.96rem;
+  }
+}
+
+h2 {
+  font-size: 1.953em;
+  font-weight: normal;
+  line-height: 1.250;
+  margin-top: 3.052rem;
+  margin-bottom: 1rem;
+}
+
 h3 {
-  font-size: 1.4em;
+  font-size: 1.563em;
+  font-weight: normal;
+  line-height: 1.250;
+  margin-top: 2.441rem;
+  margin-bottom: 1rem;
 }
 
-a h2 {
-  margin-top: 0.83em;
+h4 {
+  font-size: 1.25em;
+  font-weight: normal;
+  margin-top: 2.441rem;
+  margin-bottom: 1rem;
+}
+
+h1 + h2,
+h1 + h3,
+h1 + h4,
+h2 + h3,
+h2 + h4,
+h3 + h4 {
+  margin-top: 1rem;
+}
+
+hr {
+  border: none;
+  height: 1px;
+  background-color: #{Css.darken20 Css.backgroundColor};
+}
+
+img {
+  border-style: none;
+  max-width: 100%;
 }
 
-/*
-  Useful for narrow screens, such as mobiles. Documentation often contains URLs
-  which would otherwise force the page to become wider, and force creation of
-  horizontal scrollbars. Yuck.
-*/
 p {
-  word-wrap: break-word;
+  font-size: 1em;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
 }
 
-/* Tables */
 table {
-  /* Seemingly necessary for the ".col table" rule above to work */
-  display: block;
+  border-bottom: 1px solid #{Css.darken20 Css.backgroundColor};
+  border-collapse: collapse;
+  border-spacing: 0;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  width: 100%;
 }
 
 td, th {
-  padding: 0.4em;
+  text-align: left;
+  padding: 0.41em 0.51em;
 }
 
-tbody tr:nth-child(odd) {
-  background-color: #eee;
```

Comment:
Hm, actually that doesn't work so well since there's no padding on the left hand side really, does it? Come to think of it the current one is actually perfectly readable as is, IMO. Shall we leave this out after all?